README.md

```diff
@@ -26,7 +26,7 @@ The Arduino Create Agent is a single binary that will sit on the traybar and wor
 Get the [latest version](https://github.com/arduino/arduino-create-agent/releases) of the Agent for all supported platforms or complete the [Getting Started](https://create.arduino.cc/getting-started/plugin/welcome).
 
 ## Apple M1 support
-At the moment the new Apple Silicon Macs released in November 2020, like the new [MacBook Pro 13"](https://www.apple.com/macbook-pro-13/), [MacBook Air](https://www.apple.com/macbook-air/) and [Mac mini](https://www.apple.com/mac-mini/) models with the [Apple M1](https://www.apple.com/mac/m1/) chip are currently NOT supported by the Create Agent.
+At the moment the new Apple Silicon Macs released in November 2020, like the [MacBook Pro 13"](https://www.apple.com/macbook-pro-13/), [MacBook Air](https://www.apple.com/macbook-air/) and [Mac mini](https://www.apple.com/mac-mini/) models with the [Apple M1](https://www.apple.com/mac/m1/) chip do not have native builds. But the Arduino Agent performs well in Rosetta 2 mode.
```

Comment:
```suggestion
At the moment the new Apple Silicon Macs released in November 2020, like the [MacBook Pro 13"](https://www.apple.com/macbook-pro-13/), [MacBook Air](https://www.apple.com/macbook-air/) and [Mac mini](https://www.apple.com/mac-mini/) models with the [Apple M1](https://www.apple.com/mac/m1/) chip do not have native builds. But the Arduino Agent performs well in [Rosetta 2 mode](https://support.apple.com/en-us/HT211861).
```

Adding a reference link might be helpful to readers who are not familiar with what "Rosetta 2 mode" is. From some quick searching, this URL seems to be a reasonably canonical reference.

Since it seems to be handled automagically by macOS, I can imagine some users have installed already but don't remember doing so. This might lead users having an unrelated problem with Arduino Cloud to think some special configuration is needed to attain this mode in order to get the agent working on their M1 Mac. I think the linked page will make the situation clear to them.